bin/travis

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+set -e
+
+sbt clean scalariformFormat test:scalariformFormat
+git diff --exit-code || (
```

Comment:
You can remove all these lines and use `sbt validateCode` instead. This way, most of this file would be unnecessary since it will only call sbt tasks. Instead, you can just remove it and add the following line to the `.travis.yml` `script` section:

```yaml
- sbt ++$TRAVIS_SCALA_VERSION mimaReportBinaryIssues
```

See #24 for more context.